ansible/manager-part-3.yml

```diff
@@ -34,13 +34,67 @@
         group: root
         mode: 0755
 
+# With this play, we can decouple the pull of the required images from the
+# subsequent roles (traefik, manager, netbox). This makes problems resulting
+# from image pulls more visible in the CI.
+- name: Pull manager images
+  hosts: testbed-manager.testbed.osism.xyz
+  connection: local
+  gather_facts: true
+
+  vars:
+    docker_registry: index.docker.io
+    docker_registry_ansible: osism.harbor.regio.digital
+    docker_registry_netbox: osism.harbor.regio.digital
```

Comment:
Can we have one single location where these variables are defined? Having ever more locations duplicating the same definitions isn't really manageable.